src/frameworks/uikit/ui_nib.rs

```diff
@@ -170,8 +170,15 @@ pub const CLASSES: ClassExports = objc_classes! {
 /// return [nib instantiateWithOwner:[UIApplication sharedApplication]
 ///                     optionsOrNil:nil];
 /// ```
-pub fn load_main_nib_file(env: &mut Environment, _ui_application: id) {
-    let path = env.bundle.main_nib_file_path();
+/// Returns true if the main nib file was loaded, false if not.
+pub fn load_main_nib_file(env: &mut Environment, _ui_application: id) -> bool {
+    let path = match env.bundle.main_nib_file_path() {
+        Some(path) => path,
+        None => {
+            // The application doesn't have a main nib file.
```

Comment:
Strangely enough, this isn't sufficient. The newest version of Crash Bandicoot Nitro Kart 3D has an NSMainNibFile key but no actual file by that name. I have no idea why this works.

I already wrote code to handle that case, so maybe I should just commit what I have? Sorry about the duplicated effort.

Reply:
(Perhaps the main nib file is only loaded if a delegate class name isn't specified…? The documentation isn't clear.)